http.echo/README.md

```diff
@@ -3,16 +3,39 @@ Listens on http port `8080` and will echo back whatever is sent to the server.
 Listens on https port `9090` and will echo back whatever is sent to the server.
 
 ### Requirements
- - Docker 20.10+
+- Kubernetes (e.g. Docker for Desktop)
+- kubectl
+- helm 3.0+
+- netcat
+
+### Install zilla engine
 
-### Start zilla engine
 ```bash
-$ docker stack deploy -c stack.yml example
-Creating network example_net0
-Creating service example_zilla
+$ helm install zilla-http-echo . --namespace zilla-http-echo --create-namespace
```

Comment:
Is the rationale for using a distinct namespace per example so that more than one example can be deployed at once?

Then only the example-specific port forwarding would be needed to make it work from localhost?

Reply:
Yep, exactly.